qiita_pet/handlers/api_proxy/studies.py

```diff
@@ -111,7 +112,7 @@ def study_get_req(study_id, user_id):
     # Study.has_access no_public=True, will return True only if the user_id is
     # the owner of the study or if the study is shared with the user_id
     study_info['has_access_to_raw_data'] = study.has_access(
```

Comment:
Study.has_access no_public=True, will return True only if the user_id is
the owner of the study or if the study is shared with the user_id

This comment should be appended to include study.public_raw_download.

Reply:
k